xml/System.Diagnostics/DistributedContextPropagator.xml

```diff
@@ -52,9 +54,19 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
+        <summary>returns the default propagator object which Current property will be initialized with.</summary>
         <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+CreateDefaultPropagator will create a propagator instance that can inject and extract the headers with field names "tracestate",
+            "traceparent" of the identifiers which are formatted as W3C trace parent, "Request-Id" of the identifiers which are formatted as a hierarchical identifier.
+            The returned propagator can inject the baggage key-value pair list with header name "Correlation-Context" and it can extract the baggage values mapped to header names "Correlation-Context" and "baggage".
```

Comment:
```suggestion
            The returned propagator can inject the baggage key-value pair list with header name "Correlation-Context", and it can extract the baggage values mapped to header names "Correlation-Context" and "baggage".
```